inc/dynamic.h

```diff
@@ -1,55 +0,0 @@
-/* Vague support for several different shared library formats */
-/* Everyone but the mac uses a varient of dlopen, while the mac uses NSAddImage */
-/* Under cygwin the shared libs have very strange names which bare little */
-/*  resemblance to what we expect. GNU creates a .la file that tells us what */
-/*  the name should be, so indirect through that */
```

Comment:
When/how did this fall out of use?

Reply:
A combination of getting rid of runtime dynamic linking (with dlopen) and getting rid of plugins